handler-host.go

```diff
@@ -66,12 +66,19 @@ type HostHandler struct {
 	// will attempt to read this directory for any existing files and
 	// send them to the host.
 	CacheFileLocation string
-	db *sql.DB
+	db                *sql.DB
 
 	// List of filenames/Ids that are currently being sent to the server,
 	// so they do not get sent again.
 	currentlySending   []string
 	currentlySendingMu sync.RWMutex
+
+	// Channel to stop processing
+	die           chan bool
```

Comment:
Why are there so many channels? Can just use one if the only intention is to stop things.

Reply:
do channels send to all subscribers?

Reply:
i guess i can just pass in x values instead of passing values to the channels...